<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mRayMainWindow</class>
 <widget class="QMainWindow" name="mRayMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mRay</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="ConnectDisconnect">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_5"/>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButtonDisconnect">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="2">
        <widget class="QGroupBox" name="ConnectionStatusGB">
         <property name="title">
          <string>connection status</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="ConnectionStatus"/>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButtonConnect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <layout class="QVBoxLayout" name="verticalLayout_6"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="MainWindowWidgets">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="MainWindowSwitches">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="ReadWrite">
          <property name="title">
           <string>read/write</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>160</width>
             <height>80</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="ReadWriteSwitch">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ActiveSet">
          <property name="title">
           <string>active set</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>160</width>
             <height>80</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="ActiveSetSwitch">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="SelectedSet">
          <property name="title">
           <string>selected set</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>216</width>
             <height>80</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="SelectedSetSwitch">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionConnection_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionSignals"/>
    <addaction name="actionParameters"/>
    <addaction name="actionGenerate_Code"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Instruments"/>
    <addaction name="actionOpen_Plotter"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuProject"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSignals">
   <property name="text">
    <string>Signals...</string>
   </property>
  </action>
  <action name="actionParameters">
   <property name="text">
    <string>Parameters...</string>
   </property>
  </action>
  <action name="actionConnection_Settings">
   <property name="text">
    <string>Connection Settings...</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to ECU</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionOpen_Instruments">
   <property name="text">
    <string>Open Instruments...</string>
   </property>
  </action>
  <action name="actionOpen_Plotter">
   <property name="text">
    <string>Open Plotter</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionGenerate_Code">
   <property name="text">
    <string>Generate Code...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
